src/component/Admin/Settings/Captcha/CapCaptcha.tsx

```diff
@@ -0,0 +1,103 @@
+import { Trans, useTranslation } from "react-i18next";
+import { FormControl, Link, Stack } from "@mui/material";
+import SettingForm from "../../../Pages/Setting/SettingForm.tsx";
+import { DenseFilledTextField } from "../../../Common/StyledComponents.tsx";
+import * as React from "react";
+import { NoMarginHelperText } from "../Settings.tsx";
+
+export interface CapCaptchaProps {
+  values: {
+    [key: string]: string;
+  };
+  setSettings: (settings: { [key: string]: string }) => void;
+}
+
+const CapCaptcha = ({ values, setSettings }: CapCaptchaProps) => {
+  const { t } = useTranslation("dashboard");
+  return (
+    <Stack spacing={3}>
+      <SettingForm title={t("settings.capInstanceURL")} lgWidth={5}>
+        <FormControl fullWidth>
+          <DenseFilledTextField
+            value={values.captcha_cap_instance_url}
+            onChange={(e) =>
+              setSettings({
+                captcha_cap_instance_url: e.target.value,
```

Comment:
You need to add new setting keys at https://github.com/cloudreve/frontend/blob/master/src/component/Admin/Settings/Settings.tsx